code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -60,6 +62,22 @@
 	user << browse(t1, "window=airlock_electronics")
 	onclose(user, "airlock")
 
+/obj/item/weapon/airlock_electronics/attackby(obj/item/C, mob/user)
+	if(istype(C,/obj/item/device/debugger))
+		switch(input(user, "Would you like to change the wireless ID or the ID tag?", "Airlock Selection") as null|anything in list("Wireless ID","ID Tag"))

```

Comment:
You could make it 
```suggestion
		switch(alert(user, "Would you like to change the wireless ID or the ID tag?", "Airlock Selection", "Wireless ID", "ID Tag")
```

Reply:
Not needed. You don't type it, the input in list makes it a selection.